examples/eventlisteners/cel-eventlistener-interceptor.yaml

```diff
@@ -8,7 +8,10 @@ spec:
     - name: cel-trig
       interceptor:
         cel:
-          filter: "headers.match('X-GitHub-Event', 'push')"
+          filter: "headers.match('X-GitHub-Event', 'pull_request')"
+          overlays:
+          - key: extensions.truncated_sha
+            expression: "truncate(body.pull_request.head.sha, 7)"
```

Comment:
Not a blocker, but one thing I'm worried about here is that there is now multiple ways to do value selection in Triggers: CEL, gjson, and jsonPath. I don't know if we can necessarily do better in this case, but we'll probably want to

Reply:
This is a tricky one, CEL provides more than just extracting values, via the extension functions, lots of regularly used functionality can be made available (e.g. base64).

I have no particular attachment to CEL, but the functionality it provides isn't available in the alternatives?